app/src/main/java/ch/hslu/refashioned/ui/color/MultiGradientFactory.java

```diff
@@ -30,7 +30,7 @@ public Color getColor(final int value) {
         return this.gradientFactories.stream()
                 .filter(g -> g.getMin() <= value && g.getMax() >= value)
                 .findAny()
-                .get()
+                .orElseGet(() -> new GradientFactory(0, 0, Color.valueOf(Color.BLACK), Color.valueOf(Color.BLACK)))
```

Comment:
Hier wäre eine ConstantFactory zu verwenden. Diese implementation gäbe  bei Aufruf von getColor eine Division by zero exception.

Reply:
gelöst mit "fallback" anstelle einer ConstantFactory